.github/workflows/gh-pages.yml

```
@@ -0,0 +1,49 @@
+name: github pages
+
+on:
+  push:
+    branches:
+      - master
+      - docs_deploy
+
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
```

I think you can do something like this:

```suggestion
      - run: sudo apt-get install pandoc
      - uses: actions/checkout@v2
```